features/admin/invite_supplier_contributor.feature

```diff
@@ -16,25 +16,3 @@ Scenario Outline: Correct users can invite a contributors to a supplier account
   Examples:
     | role                    | link-text                               |
     | admin                   | Edit supplier accounts or view services |
-    | admin-ccs-category      | Edit suppliers and services             |
```

Comment:
Do we still need this table?

Reply:
resolved in 5be917f8517a1de127ea58e24906a8d7ca8f52a3